R/utils.R

```diff
@@ -0,0 +1,16 @@
+execute_aws_cmd <- function(args) {
+  error_log <- paste0(tempdir(), "error.log")
```

Comment:
We should remove the error log file afterward, if there's no error.

Reply:
Fixed as part of [feature/KBI-68](https://github.com/precision-analytics/precisely.aws/blob/96a3f8e5a9ad74ffc7eb8a90d92a94d0429087b1/R/utils.R).